.github/CODEOWNERS

```diff
@@ -331,9 +331,6 @@ packages/react-components/react-colorpicker-compat @microsoft/cxe-red @sopranopi
 packages/react-components/react-nav-preview @microsoft/cxe-red @microsoft/xc-uxe @mltejera
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32190&commitId=9a5d65fe8c8397aeded24bd6951f9f7ba31f4aa2&env=prod&clientType=FLUENTUI)


  <details><summary><b>Card Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.size.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Filled.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled.selected.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>